src/components/data-management/ProjectDetails.jsx

```diff
@@ -301,6 +302,7 @@ const ProjectDetails = ({ width, height }) => {
             massEdit
           >
             <SpeciesSelector
+              testID='fill-species-select'
```

Comment:
We have refactored the test ids into a common file `src/utils/integrationTestConstants.js`. You can see how test id is implemented here : https://github.com/biomage-ltd/ui/blob/master/src/components/data-management/ProjectMenu.jsx#L73

Reply:
you'll also need to call this attribute `test-data-id`